crates/sb_workers/events.rs

```diff
@@ -24,26 +22,25 @@ pub struct IncomingEvent {
 }
 
 #[op]
-async fn op_event_accept(state: Rc<RefCell<OpState>>) -> Result<RawEvent, AnyError> {
-    let mut rx = {
+async fn op_event_accept(state: Rc<RefCell<OpState>>) -> Result<RawEvent, Error> {
+    let rx = {
         let mut op_state = state.borrow_mut();
-        op_state.take::<mpsc::UnboundedReceiver<WorkerEvents>>()
+        op_state.try_take::<mpsc::UnboundedReceiver<WorkerEvents>>()
     };
+    if rx.is_none() {
+        bail!("events worker receiver not available")
+    }
+    let mut rx = rx.unwrap();
 
-    let data = rx.try_recv();
```

Comment:
@andreespirela Changed this to use async `recv` instead of `try_recv`. Previously, this was processing the empty values causing CPU spikes.